src/api.ts

```diff
@@ -115,3 +115,37 @@ export class TestRuns {
     }
   }
 }
+
+export class Jobs {
+  private api: AxiosInstance;
+  private username: string;
+
+  constructor(opts: { username: string; accessKey: string; region: Region }) {
+    this.api = axios.create({
+      auth: {
+        username: opts.username,
+        password: opts.accessKey,
+      },
+      baseURL: apiURLMap.get(opts.region),
+    });
+    this.username = opts.username;
+  }
+
+  async updateStatus(jobId: string, passed: boolean) {
+    try {
+      await this.api.put<void>(`/rest/v1/${this.username}/jobs/${jobId}`, {
+        passed,
+      });
+    } catch (e: unknown) {
+      if (isAxiosError(e)) {
+        debug(
+          'Unexpected http error while reporting test run data: %s',
+          e.message,
+        );
+      } else {
+        debug('Unexpected error while reporting test run data', e);
+      }
```

Comment:
💭 Without running this code, it's not immediately obvious to me what benefit this debug-logging offers here, considering that `updateJobStatus(jobId, passed)` catches the error later and logs it out to the console 🤔

Reply:
Right, good point.